python/cugraph/cugraph/tests/mg/test_mg_louvain.py

```diff
@@ -103,26 +110,19 @@ def uddaskGraphFromDataset(request, dask_client):
 # @pytest.mark.skipif(
 #    is_single_gpu(), reason="skipping MG testing on Single GPU system"
 # )
-def test_mg_louvain_with_edgevals(daskGraphFromDataset):
-    # FIXME: daskGraphFromDataset returns a Directed graph, which Louvain is
-    # currently accepting. In the future, an MNMG symmeterize will need to
-    # be called to create a Graph for Louvain.
-    parts, mod = dcg.louvain(daskGraphFromDataset)
-
-    # FIXME: either call Nx with the same dataset and compare results, or
-    # hardcode golden results to compare to.
-    print()
-    print(parts.compute())
-    print(mod)
-    print()
+def test_mg_louvain_with_edgevals_directed_graph(daskGraphFromDataset):
+    # Directed graphs are not supported by Louvain and an exception should be
+    # raised
+    with pytest.raises(Exception):
```

Comment:
This should be changed to something more specific too (`ValueError`). Catching `Exception` will catch everything and will hide bugs.

Reply:
Addressed